tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3718 @@
+                  type: string
+                  description: The coefficient used for calculation of initial rewards.
+                decayBalancingConstantExponent:
+                  type: integer
+                  description: DecayBalancingConstantExponent is the exponent used for calculation of the initial reward.
+                decayBalancingConstant:
+                  type: string
+                  description: An integer approximation calculated based on chosen DecayBalancingConstantExponent. 
+                poolCoefficientExponent:
+                  type: integer
+                  description: PoolCoefficientExponent is the exponent used for shifting operation in the pool rewards calculations.
+              required:
+                - profitMarginExponent
+                - bootstrappingDuration
+                - manaShareCoefficient
+                - decayBalancingConstantExponent
+                - decayBalancingConstant
+                - poolCoefficientExponent
+            targetCommitteeSize:
+              type: integer
+              description: The target size of the committee.
+            chainSwitchingTreshold:
+              type: integer
+              description: The number of heavier slots that a chain needs to be ahead of the current chain to be considered for switching.
+          required:
+            - version
+            - networkName
+            - bech32Hrp
+            - tokenSupply
+            - storageScoreParameters
+            - workScoreParameters
+            - manaParameters
+            - genesisSlot
+            - genesisUnixTimestamp
+            - slotDurationInSeconds
+            - slotsPerEpochExponent
+            - stakingUnbondingPeriod
+            - validationBlocksPerSlot
+            - punishmentEpochs
+            - livenessThresholdLowerBound
+            - livenessThresholdUpperBound
+            - minCommittableAge
+            - maxCommittableAge
+            - epochNearingThreshold
+            - congestionControlParameters
+            - versionSignalingParameters
+            - rewardsParameters
+            - targetCommitteeSize
+            - chainSwitchingTreshold
+      required:
+        - startEpoch
+        - parameters
+
+    ValidatorsResponse:
+      description: Returns a paginated list of all registered validators ready for the next epoch and indicates if they were active recently (are eligible for committee selection).
+      properties:
+        validators:
+          type: array
+          items:
+            oneOf:
+              - $ref: '#/components/schemas/Validator'
+        pageSize:
+          type: integer
+          description: The number of validators returned per one API request with pagination.
+        cursor:
+          type: string
+          description: The cursor that needs to be provided as cursor query parameter to request the next page. Cursor is absent if the last page is returned.
+
+    ManaRewardsResponse:
```

Comment:
Need to add `LatestCommittedEpochPoolRewards`
https://github.com/iotaledger/iota.go/pull/651